theme/templates/resource-landing-page/title-section.html

```diff
@@ -113,6 +113,13 @@
                         <span>A collection must contain at least one resource before it can be published, public or discoverable.</span>
                     {% endif %}
                 {% endif %}
+
+                {% if cm.non_preferred_path_names %}
```

Comment:
Just something to ponder, I think it makes sense either way...
I wonder if it makes more sense to have this notification show near the filebrowser element, near the other file-related messages:
![image](https://user-images.githubusercontent.com/17934193/203143399-fab87a29-819a-4fa5-8429-6b1b6e89e762.png)

Reply:
Since this is something we want the user to fix, I think displaying this message at the top of the page is probably better.

Reply:
sounds good